createClient.js

```diff
@@ -33,6 +33,22 @@ function createClient(options) {
     })
   });
 
+
+  client.socket.write(Buffer.from("02","hex")); // This initialises a BNFTP file download conversation
```

Comment:
no, you need to create a client2 (client2 = new Client2() ; client2.connect() , etc) and at the stop Client2 = require('./client2')